src/System.IO.Compression/src/Interop/Interop.clrcompression.cs

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.IO.Compression;
+
+internal static partial class Interop
+{
+    internal static partial class clrcompression
+    {
+        [DllImport("clrcompression.dll")]
```

Comment:
All of these functions are exported by zlib, right?  Can you pull the library name out into a constant in src\Common\src\Interop\Windows\Interop.Libraries.cs?  Assuming these are all the same exports regardless of platform, that'll let us define the name differently in src\Common\src\Interop\Unix\Interop.Libraries.cs, and it should theoretically "just work"... theoretically ;)